benchmark-service/benchmark/benchmark.py

```diff
@@ -240,15 +316,23 @@ async def update_commits(app):
 
 
 async def get_commit(app, sha):  # pylint: disable=unused-argument
+    log.info('in get_commit')
     github_client = app['github_client']
     batch_client = app['batch_client']
     gs_reader = app['gs_reader']
 
-    file_path = f'{BENCHMARK_RESULTS_PATH}/{sha}.json'
+    file_path = f'{BENCHMARK_RESULTS_PATH}/0-{sha}.json'
     request_string = f'/repos/hail-is/hail/commits/{sha}'
-
     gh_commit = await github_client.getitem(request_string)
 
+    message = gh_commit['commit']['message']
+    idx = message.index('#')
+    commit_id = message[idx: idx + 5]
```

Comment:
What is the commit_id? Is this the PR number?

Reply:
yeah